### ISSUE TITLE
mistake in code

### ISSUE THREAD

**Issue by Simon-Smith-NKU** [haonan-yuan/EAGLE#3]
https://github.com/haonan-yuan/EAGLE/blob/17b72758a7fa1c7e075c78297c8540d916852b12/EAGLE/runner.py#L366

There's a significant error here.

Source code：
        cvae_loss = self.loss_cvae(recon, x_flatten, mu, log_std) / (
            len(embeddings[0] * len(embeddings[0]))
        )
embeddings[0] * len(embeddings[0]) equals to constant_K *embeddings[0], this makes no sense
Thus,len(embeddings[0] * len(embeddings[0])) == len(embeddings[0]) == num_node.

Correct code：
       cvae_loss = self.loss_cvae(recon, x_flatten, mu, log_std) / len(x_flatten)
`x_flatten:(args.n_factors * t * num_nodes, delta_d) `
thus, len(x_flatten) == num_environment`

Wrong cvae_loss leads to wrong total loss,,as follows:
 `loss = main_loss + alpha * var_loss + beta * cvae_loss`
